### PR TITLE
feat: set focus on first form element CB-124

### PR DESCRIPTION
**Pull request by dkrupenya**

### CODE REVIEW

File: webapp/packages/core/src/blocks/InFocus.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+type InFocusProps = {
+  children: ReactElement;
+}
+
+export function InFocus({ children }: InFocusProps) {
+  const isFirstRender: MutableRefObject<boolean> = useRef(true);
+  const childRef: MutableRefObject<HTMLElement | null> = useRef(null);
+
+  useEffect(() => {
```
Comment (Wroud) [inline webapp/packages/core/src/blocks/InFocus.tsx]: useLayoutEffect better for purpose like this

File: webapp/packages/core/src/blocks/InFocus.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        }
+
+      }
+    }
+  });
+  const childElement = React.Children.only(children);
+  return React.cloneElement(
+    childElement,
+    { ref: (el: HTMLElement) => childRef.current = el }
+  );
```
Comment (Wroud) [inline webapp/packages/core/src/blocks/InFocus.tsx:30]: first child can be not HTML element, so maybe better to wrap `<div {...props} ref={childRef}>{children}</div>`

File: webapp/packages/core/src/blocks/InFocus.tsx (hunk 1 above)
Comment (dkrupenya) [reply webapp/packages/core/src/blocks/InFocus.tsx]: fixed

File: webapp/packages/core/src/blocks/InFocus.tsx (hunk 2 above)
Comment (dkrupenya) [reply webapp/packages/core/src/blocks/InFocus.tsx:30]: just don't use InFocus around non-html components
The Idea is to have inFocus as HOC that dosn't append something to DOM

Comment (Wroud) [reply webapp/packages/core/src/blocks/InFocus.tsx:30]: yep I understand that idea, in this case, may be better to replace InFocus with hook that returns ref, that we can pass to HTML element

**Pull request merged**